src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json

```diff
@@ -6,8 +6,8 @@
   "testerName":"Deepika Singh",
   "testerStaffId":"d0b09235-7234-4bed-b628-e4c2a1fcb414",
   "testerEmailAddress":"deepikasingh@dvsagov.onmicrosoft.com",
-  "testStartTimestamp":"2019-12-12T10:41:53.012Z",
-  "testEndTimestamp":"2019-12-12T10:42:58.970Z",
+  "testStartTimestamp":"2020-12-12T10:41:53.012Z",
```

Comment:
Why were the dates updated with 2020 not 2021?

Reply:
The tests date were outdated hence updated.